ui/src/ScholarReader.tsx

```diff
@@ -313,6 +321,28 @@ class ScholarReader extends React.PureComponent<ScholarReaderProps, State> {
         if (symbols.length >= 1) {
           const mathMl = await api.mathMlForArxivId(this.props.paperId.id);
           this.setMathMl(mathMl);
+
+          /**
+           * Build a mapping from symbol to all possible parent matches
+           */
+          const symbolMatchSet: SymbolMatchSet = {};
```

Comment:
Can this be named something like `symbolMatches` or `symbolMatchSets`? The data structure is a dictionary of sets, though the name makes me wonder if it's a set of matches.

Reply:
Yes I will rename it for clarity :)